tests/components/test_websocket_api.py

```diff
@@ -300,8 +300,14 @@ def test_get_panels(hass, websocket_client):
     assert msg['id'] == 5
     assert msg['type'] == wapi.TYPE_RESULT
     assert msg['success']
-    assert msg['result'] == {url: panel.as_dict() for url, panel
-                             in hass.data[frontend.DATA_PANELS].items()}
+    assert msg['result'] == [{
```

Comment:
This breaks the frontend. It expects an object and not a map in https://github.com/home-assistant/home-assistant-polymer/blob/master/src/layouts/partial-panel-resolver.html#L82

Reply:
I've reverted this change.